common/config/rush/pnpm-lock.yaml

```diff
@@ -6338,17 +6315,17 @@ packages:
       node: '>= 0.4'
     resolution:
       integrity: sha512-exHJeq6kBKj58mqGyTQ9DFvrZC/eR6OwxzoM9YRoGBqrXYonaFyGiFMuc9VZrXf7DarreEwMpurG3dd+CNyW5w==
-  /object.entries/1.1.0:
+  /object.entries/1.1.1:
     dependencies:
       define-properties: 1.1.3
-      es-abstract: 1.16.3
+      es-abstract: 1.17.0-next.1
```

Comment:
This is a wee bit suspicious to me. object.entries declares the constraint `^1.13.0` which should not match anything with a pre-release tag. The [semver calculator](https://semver.npmjs.com/) agrees with me.

Reply:
Thanks for reviewing this diff, I usually don't review it this closely.  I agree this is suspicious and I'll investigate.

Reply:
I'm not sure what happened last time, but the version of `es-abstract` has been updated to `1.17.0` in an earlier update to `pnpm-lock.yaml`.